credential/cargo-credential-1password/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "cargo-credential-1password"
 version = "0.4.0"
 edition.workspace = true
 license.workspace = true
+rust-version = "1.70.0"  # MSRV:3
```

Comment:
From https://github.com/rust-lang/cargo/pull/12381#issuecomment-1698283115

> Just realized, with this pull request and #12395, it is inevitable that every time a Rust release comes out, `cargo-util` and `crates-io` will be forced to release a version due to the bump of `package.rust-version`, even when there is no actual code change involved.
> 
> Is that something we've considered? It feels like a bit unnecessary.

What we could do is swap `workspace.rust-version` so it is `MSRV:3` and so these packages will be "untouched" despite getting MSRV bumps

Reply:
@weihanglo any thoughts on whether `workspace.package.rust-version` should track the MSRV for internal packages (MSRV:1) or user-facing packages (MSRV:3)?